scm/driver/stash/webhook.go

```diff
@@ -127,6 +143,16 @@ type pullRequestHook struct {
 	Date        string `json:"date"`
 	Actor       *user  `json:"actor"`
 	PullRequest *pr    `json:"pullRequest"`
+	// only in pr:from_ref_updated
+	PreviousFromHash string `json:"previousFromHash"`
+	// only in pr:modified
+	PreviousTarget struct {
+		ID              string `json:"id"`              // "refs\/heads\/master"
+		DisplayID       string `json:"displayId"`       // "master"
+		Type            string `json:"type"`            // "BRANCH"
+		LatestCommit    string `json:"latestCommit"`    // "860c4eb4ed0f969b47144234ba13c31c498cca69"
```

Comment:
Only fields `. PreviousTarget` and `.PreviousTarget.LatestCommit` are currently used. Should I remove everything else from here?

Additionally, if not, should I keep the examples of the data in comments or should I drop those?